extensions/blocks/podcast-player/components/podcast-player.js

```diff
@@ -73,8 +75,13 @@ export class PodcastPlayer extends Component {
 		this.setState( { currentTrack: track } );
 		this.setAudioSource( trackData.src );
 
-		// Read that we're loading the track and its description. This is dismissible via ctrl on VoiceOver.
-		/* translators: %s is the track title. It describes the current state of the track as "Loading: [track title]" */
+		/*
+		 * Read that we're loading the track and its description. This is
+		 * dismissible via ctrl on VoiceOver.
+		 *
+		 * translators: %s is the track title. It describes the current state of the
+		 * track as "Loading: [track title]".
+		 */
```

Comment:
Have you tested if this comment annotation gets picked up as expected?

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/15281/commits/116bc2b09cbb167da38709b588dc5e4296ed4b5f